common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -15,10 +15,11 @@
 import net.minecraft.network.chat.MutableComponent;
 import net.minecraft.network.chat.Style;
 import net.minecraft.network.chat.TextColor;
+import net.minecraft.network.chat.TextComponent;
 import org.jetbrains.annotations.Nullable;
 
 public class ComponentUtils {
-    public static String getFormatted(Component component) {
+    public static String getFormatted(FormattedText component) {
```

Comment:
What's this change? I can't see this fit on, also what is a FormattedText and how does it relate to Component?

Reply:
the Component interface extends the FormattedText interface.

This was a change that I was not really fond of as well, as I did not write this original functionality. I'm not really sure how it works but as far I know the method is supposed to return the §-color-coded string from a Component. I needed a way to create a Component from a FormattedText, and found that this method doesn't use Component-only behavior, so I changed the parameter type to FormattedText instead.
I used this in the formattedTextToComponent method which goes ahead and converts every FormattedText supplied from splitText back to Component by calling this getFormatted function.
This was the change that allowed me to feed back a List<Component> in the ItemTooltipRenderEvent.Pre instead of the vanilla internal-wizardry I was doing.
I maybe should've also changed the name of the parameter. (should I?)

(also, If you know of a better way to go from FormattedText to Component, I'll gladly use that)

Reply:
If you change the type, please change the argument name too. Also, I think it is a reasonable change.

Reply:
But I will wait for magicus' comment before merging.